pkg/network/ovn_kubernetes.go

```diff
@@ -2547,5 +2519,148 @@ func annotateNodeDaemonset(kubeClient kubernetes.Interface) error {
 	}); err != nil {
 		return err
 	}
+
```

Comment:
Did you add this empty line intentionally? If not, we can get rid of this and not make part of this PR(nit).